.htaccess

```diff
@@ -1,6 +1,18 @@
 <IfModule mod_headers.c>
     Header set Access-Control-Allow-Origin "*"
 </IfModule>
+<IfModule mod_mime.c>
+    AddType font/ttf .ttf
+    AddType font/woff .woff
+    AddType font/woff2 .woff2
+    AddType font/otf .otf
+    AddType application/octet-stream .eot
+    AddType application/x-apple-aspen-config .mobileconfig
+    AddType image/svg+xml .svg
+    AddType application/javascript .js 
+
+    AddCharset UTF-8 .js
+</IfModule>
 <IfModule mod_rewrite.c>
     RewriteEngine on
 
```

Comment:
RewriteRule on l.19
Should we have `build.sh` and `.htaccess` give a 404?

Reply:
yes good idea